data/wanderers.txt

```diff
@@ -349,7 +349,222 @@ phrase "friendly wanderer"
 		"ecosystems."
 		"biospheres."
 
+phrase "friendly wanderer"
+	word
+		"You are "
+	word
+		"a welcome "
+		"an honored "
+	word
+		"[visitor, traveller] "
+		"guest "
+	word
+		""
+		"to us"
+		"to our people"
+		"to the Wanderers"
+	word
+		", "
+		" - "
+	word
+		"so long as "
+		"as long as "
+		"[providing, given] "
+	word
+		"you respect our ways."
+		"you respect our worlds."
+		"you respect our ecosystems."
+		"you show respect for our ways."
+		"you show respect for our worlds."
+		"you show respect for our ecosystems."
+		"you live sustainably."
+		"you [consume, devour] no more than you need."
+		"you cause no harm."
+		"you come in peace."
 
+phrase "friendly wanderer"
+	word
+		"We mean "
+		"We [intend, plan] "
+		"We come "
+	word
+		"to "
+		" only to "
+		"[simply, solely] to "
+	word
+		"[repair, fix] "
+		"restore "
+		"heal "
+		"[shape, remake] "
+		"[transform, renew] "
+		"revitalize "
+		"take care of "
+		"breathe [life, hope] "
+	word
+		"what has been "
+		"what others have "
+		"what those who came before have "
+	word
+		"[harmed, abused]."
+		"broken."
+		"[brutalized, mistreated]."
+		"[wasted, spent]."
+		"killed."
+		"rendered [lifeless, barren]."
+		"ruined."
+		"[consumed, devoured]."
+		"exploited."
+		"poisoned."
+		"wrecked."
```

Comment:
Given that these all end in a period you could add
```
	word
		"."
```
at the very end of those phrase